presto-hive/src/main/java/io/prestosql/plugin/hive/security/SystemTableAwareAccessControl.java

```diff
@@ -40,6 +40,90 @@ public SystemTableAwareAccessControl(ConnectorAccessControl delegate)
+    {
+        delegate.checkCanDropSchema(transactionHandle, identity, schemaName);
+    }
+
+    @Override
+    public void checkCanRenameSchema(ConnectorTransactionHandle transactionHandle, ConnectorIdentity identity, String schemaName, String newSchemaName)
+    {
+        delegate.checkCanRenameSchema(transactionHandle, identity, schemaName, newSchemaName);
+    }
+
+    @Override
+    public void checkCanShowSchemas(ConnectorTransactionHandle transactionHandle, ConnectorIdentity identity)
+    {
+        delegate.checkCanShowSchemas(transactionHandle, identity);
+    }
+
+    @Override
+    public Set<String> filterSchemas(ConnectorTransactionHandle transactionHandle, ConnectorIdentity identity, Set<String> schemaNames)
+    {
+        return delegate.filterSchemas(transactionHandle, identity, schemaNames);
+    }
+
+    @Override
+    public void checkCanCreateTable(ConnectorTransactionHandle transactionHandle, ConnectorIdentity identity, SchemaTableName tableName)
+    {
+        delegate.checkCanCreateTable(transactionHandle, identity, tableName);
+    }
+
+    @Override
+    public void checkCanDropTable(ConnectorTransactionHandle transactionHandle, ConnectorIdentity identity, SchemaTableName tableName)
+    {
+        delegate.checkCanDropTable(transactionHandle, identity, tableName);
+    }
+
+    @Override
+    public void checkCanRenameTable(ConnectorTransactionHandle transactionHandle, ConnectorIdentity identity, SchemaTableName tableName, SchemaTableName newTableName)
+    {
+        delegate.checkCanRenameTable(transactionHandle, identity, tableName, newTableName);
+    }
+
+    @Override
+    public void checkCanShowTablesMetadata(ConnectorTransactionHandle transactionHandle, ConnectorIdentity identity, String schemaName)
+    {
+        delegate.checkCanShowTablesMetadata(transactionHandle, identity, schemaName);
+    }
+
+    @Override
+    public Set<SchemaTableName> filterTables(ConnectorTransactionHandle transactionHandle, ConnectorIdentity identity, Set<SchemaTableName> tableNames)
+    {
+        return delegate.filterTables(transactionHandle, identity, tableNames);
+    }
+
+    @Override
+    public void checkCanAddColumn(ConnectorTransactionHandle transactionHandle, ConnectorIdentity identity, SchemaTableName tableName)
+    {
+        delegate.checkCanAddColumn(transactionHandle, identity, tableName);
+    }
+
+    @Override
+    public void checkCanDropColumn(ConnectorTransactionHandle transactionHandle, ConnectorIdentity identity, SchemaTableName tableName)
+    {
+        delegate.checkCanDropColumn(transactionHandle, identity, tableName);
+    }
+
+    @Override
+    public void checkCanRenameColumn(ConnectorTransactionHandle transactionHandle, ConnectorIdentity identity, SchemaTableName tableName)
+    {
+        delegate.checkCanRenameColumn(transactionHandle, identity, tableName);
+    }
+
```

Comment:
these above methods should be already defined in `ForwardingConnectorAccessControl`, no need to have them here